debug/MVT.html

```diff
@@ -40,8 +40,8 @@ <h1>Add layer</h1>
       style: carto.basemaps.voyager,
       center: [-73.97720677029952, 40.77784846220871],
       zoom: 13,
-      dragRotate: false,
-      touchZoomRotate: false,
+
```

Comment:
Shouldn't we be explicit and have `dragRotate: true`?

Reply:
I'd prefer to reduce noise